pages/index.tsx

```diff
@@ -22,10 +23,12 @@ export type FullLocalStorage = {
 
 export default function Home() {
   const [giftData, setGiftData] = useState<FullLocalStorage[]>([]);
-  const [giftNameError, setGiftNameError] = useState<boolean>(false);
-  const [receiverError, setReceiverError] = useState<boolean>(false);
-  const [newReceiver, setNewReceiver] = useState<string>('');
-  const [newGiftName, setNewGiftName] = useState<string>('');
+  const [giftNameError, setGiftNameError] = useState(false);
+  const [receiverError, setReceiverError] = useState(false);
+  const [newReceiver, setNewReceiver] = useState('');
+  const [newGiftName, setNewGiftName] = useState('');
+  const [openModal, setOpenModal] = useState(false);
```

Comment:
Itse suosin tämän tyyppisissä boolean-muuttujissa is-alkua, eli `isModalOpen` ja `setIsModalOpen`. Toki vähän makuasia.